src/main/java/seedu/duke/flashutils/utils/Parser.java

```diff
@@ -156,9 +156,9 @@ private static long parseTimer(String timer) {
         String unit = parts[1];
 
         return switch (unit) {
-        case "second", "seconds" -> (long) (value * 1000);
-        case "minute", "minutes" -> (long) (value * 1000 * 60);
-        default -> throw new IllegalArgumentException("Unsupported time unit: " + unit);
+            case "second", "seconds" -> (long) (value * 1000);
+            case "minute", "minutes" -> (long) (value * 1000 * 60);
+            default -> throw new IllegalArgumentException("Unsupported time unit: " + unit);
```

Comment:
Does not follow the stylechecker rules, previous version was correct (as bad as it looks).

Reply:
noted! will make the changes